### ISSUE TITLE
[Bug] Harmonize normal mode and private mode home screens

### ISSUE THREAD

**Issue by s-ankur** [mozilla-mobile/fenix#7628]
TL;DR: Pls remove save to collection option in the 3 dot menu in normal mode home screen.
### Why/User Benefit/User Problem
Private Mode home screen has two tab related options:delete and share. These are accessible directly from the home screen. Normal mode home screen has a 3 dot menu with 3 options, save to collection being the additional option. There are already two additional ways to save to collection: the button and long press to select.
### What/Requirements
 Since the save to collection option in the 3 dot menu is redundant, we should remove the 3 dot menu and add the remaining options directly on the tab menu. this will make tab management much easier
### Acceptance Criteria (how do I know when I’m done?)
There are two options above the tabs in normal mode: delete and share, making normal mode home screen similar to private mode home screen
